ibm/resource_ibm_is_vpn_server.go

```diff
@@ -0,0 +1,806 @@
+			},
+			"created_at": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The date and time that the VPN server was created.",
+			},
+			"crn": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The CRN for this VPN server.",
+			},
+			"enable_split_tunneling": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Indicates whether the split tunneling is enabled on this VPN server.",
+			},
+			"health_state": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The health of this resource.- `ok`: Healthy- `degraded`: Suffering from compromised performance, capacity, or connectivity- `faulted`: Completely unreachable, inoperative, or otherwise entirely incapacitated- `inapplicable`: The health state does not apply because of the current lifecycle state. A resource with a lifecycle state of `failed` or `deleting` will have a health state of `inapplicable`. A `pending` resource may also have this state.",
+			},
+			"hostname": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Fully qualified domain name assigned to this VPN server.",
+			},
+			"href": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The URL for this VPN server.",
+			},
+			"vpn_server": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The unique identifier for this VPN server.",
+			},
+			"lifecycle_state": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The lifecycle state of the VPN server.",
+			},
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: InvokeValidator("ibm_is_vpn_server", "name"),
+				Description:  "The user-defined name for this VPN server. If unspecified, the name will be a hyphenated list of randomly-selected words. Names must be unique within the VPC this VPN server is serving.",
+			},
+			"port": &schema.Schema{
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      443,
+				ValidateFunc: InvokeValidator("ibm_is_vpn_server", "port"),
+				Description:  "The port number to use for this VPN server.",
+			},
+			"private_ips": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
```

Comment:
nothing inside this should be optional when its computed

Reply:
done.